modules/opp-architecture-installing-policyset.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * architecture/opp-architecture.adoc
+
+:_module-type: PROCEDURE
+[id="opp-architecture-installing-policyset_{context}"]
+= Installing {product-title} by using PolicySets
```

Comment:
Per our guidelines [1], we should normally follow the format for general references in most cases when referring to resources. 

So, following that, these would be written as "policy sets". 

If you really need to refer to the API object, then they need to be marked up as `PolicySet` object (assuming it's actually singular). 

So I'd recommend doing a sweep through this PR and updating most instances to "policy set" where you can, and then update to `PolicySet` object/resource/etc. wherever you need to get specific. Let me know if you have any questions on this!

[1] https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting